src/main/java/com/exactpro/th2/fix/client/fixBean/FixBean.java

```diff
@@ -98,90 +105,184 @@ public String toString() {
                 .toString();
     }
 
+    public void setConnectionType(String connectionType) {
+        if (connectionType == null || connectionType.equals("")) {
+            throw new IllegalArgumentException("connectionType must be only initiator.");
+        }
+        this.connectionType = connectionType;
+    }
+
+    public void setFileStorePath(String fileStorePath) {
+        if (fileStorePath == null || fileStorePath.equals("")) {
+            throw new IllegalArgumentException("fileStorePath must not be null or blank.");
```

Comment:
> or blank

You've only checked that it's not empty - blank (empty or consisting only of whitespaces) != empty (zero length)